packages/webtest-js/webtest-js.2.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+authors: "John Else"
+homepage: "https://github.com/johnelse/ocaml-webtest"
+bug-reports: "https://github.com/johnelse/ocaml-webtest/issues"
+dev-repo: "git://github.com/johnelse/ocaml-webtest"
+maintainer: "john.else@gmail.com"
+synopsis: "An in-browser js_of_ocaml testing framework - js_of_ocaml integration"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" ] {with-test}
+]
+
+depends: [
+  "dune" {>= "1.1"}
+  "js_of_ocaml" {>= "3.4.0"}
+  "js_of_ocaml-ppx" {>= "3.4.0"}
+  "ocaml" {>= "4.02.1"}
+  "webtest"
```

Comment:
We tend to use stricter versioning when packages are released together. Would you mind if we add the following constraint? Could you also port it upstream?

```suggestion
  "webtest" {=version}
```

Reply:
Sure! I've added this change.